coupling_components/solver_wrappers/abaqus/USR.f

```diff
@@ -434,17 +435,18 @@ SUBROUTINE UTRACLOAD(ALPHA,T_USER,KSTEP,KINC,TIME,
       FOUND  = .FALSE.
       IF (S > 1) THEN
          DO R = 1,S
-            IF (INDEX(SNAME, TRIM(SURFACEIDS(R))) > 0) THEN
+            PREPENDED = 'ASSEMBLY_' // SURFACEIDS(R)
```

Comment:
As I see it here, 'PREPENDED' is declared as an array, but here it is used as a single string?
Additionally, if SURFACEIDS(R) is allowed to be a 80-character long string, I think PREPENDED should be declared as 89-charcter long string so that in case an 80-character long string is used for SURFACEIDS(R), the PREPENDED version does not get cropped. Or is SNAME returned by Abaqus by default at most 80 characters long?

Reply:
You're right, it should be declared on a separate line as a single string (1D character array), indeed maybe with length 89 to be consistent.